programmers/난이도별/level01.서울에서_김서방_찾기/sjjyy.java

```diff
@@ -0,0 +1,23 @@
+package 난이도별.level01.서울에서_김서방_찾기;
+
+public class sjjyy
+{
+    public static String solution(String[] seoul)
+    {
+        int index = 0;
+        for (int i = 0 ; i < seoul.length ; i++)
+        {
+            if (seoul[i].equals("Kim"))
+                index = i;
```

Comment:
```suggestion
                return "김서방은 " + i + "에 있다";
```
문제 읽어보니까 무조건 한 번만 나타난다고 하네요! 그래서 바로 리턴해도 될 것 같습니다~